### ISSUE TITLE
Are you training with COCO?

### ISSUE THREAD

**Issue by MUVGuan** [yoxu515/aot-benchmark#15]
Hello!
When I use COCO as one of the Static datasets to train the pre stage, I find that it reduces the accuracy of the pre-training model tested on DAVIS17. The pre-train model trained 20000 iters by using MSRA10K、PASCAL-S、PASCAL-VOC、ECSSD achieve nearly 59% iou when test on DAVIS17-val, but after adding COCO, even train 100000 iters, its iou is only 48%.
What do you think might have caused this? Are the COCO's annotations themself not particularly accurate?
And are you training with COCO?
Thank you very much!

**Comment by z-x-yang** [comment]
According to my experiments, the VOS performance of the pretrained stage can't reflect the performance of the main stage. The pre-trained models trained with 20000 iterations always have better vos accuracy than 100000 iterations. But the latter ones have better main-training performance.

**Comment by MUVGuan** [comment]
Thanks for your reply!
There is another question. When I train the model 20000 iterations with static images, there's a huge difference in performance whether use COCO or not. If use COCO,  the performance is only 48%, but if not use the performance is 58%. Based on your experience, is that normal? Also, do I have to train with COCO?

**Comment by z-x-yang** [comment]
I didn't try to train AOT without COCO, but I suppose COCO is necessary due to its large scale.

**Comment by MUVGuan** [comment]
Ok, thank you very much!

**Comment by MUVGuan** [comment]
Sorry, I have another question. In your experiments, the COCO ground truth mask you used for pre-training, is it a binary image like I put below? Or normal semantic segmentation mask?
![图片](https://user-images.githubusercontent.com/84904990/162985715-7b29bcf4-22c1-445e-8e95-18b8eb8dba65.png)

**Comment by z-x-yang** [comment]
It should be multi-object masks.

**Comment by MUVGuan** [comment]
You mean like this one?
![图片](https://user-images.githubusercontent.com/84904990/163298599-bfcc4adb-5bc4-49a6-86d9-3ba7818403cd.png)

**Comment by z-x-yang** [comment]
Right.

**Comment by MUVGuan** [comment]
Thank you very much!
